### PR TITLE
Cleaning up a lot of things

### PR DESCRIPTION
**Pull request by magnuh**
Cleaning up:
- visual code examples
- faulty code in examples
- wrong info in wrong places
- duplicated things
- unescaped "{" / "}"
- table layouts
- typos
- document structure
-  ...

### CODE REVIEW

File: docs/outline-style.md [hunk 1]
```diff
@@ -45,40 +45,21 @@ Use `outline-none` to hide the default browser outline on focused elements.
...
-  placeholder="No focus with outline-none"
-  class="outline-none rounded-8 font-sans block border-2 border-solid pd-border-slate-900/10 pd-text-sm w-full py-4 px-6 text-slate-500 rounded-lg shadow-sm" />
-
-<input type="text"
-  placeholder="Custom focus style"
-  class="focusable focus:[--w-outline-offset:-2px] rounded-8 font-sans block border-2 border-solid pd-border-slate-900/10 pd-text-sm w-full py-4 px-6 text-slate-500 rounded-lg shadow-sm" />
+  <div class="flex flex-col sm:flex-row items-center justify-center gap-x-16 gap-y-8 px-0 sm:px-10 font-mono font-bold">
+    <input type="text" class="focus:outline s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="Default focus style" />
+    <input type="text" class="outline-none s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="No focus with outline-none" />
+    <input type="text" class="focusable focus:[--w-outline-offset:-2px] s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="Warp:s focus style" />
```
Comment (BalbinaK) [inline docs/outline-style.md:51]: What is `Warp:s` focus?

**Review by BalbinaK — APPROVED** [review]
Looks good to me! 😎

File: docs/outline-style.md (hunk 1 above)
Comment (magnuh) [reply docs/outline-style.md:51]: Well.... I am not really sure... 🤷 😄 
I haven't changed the wording here... Just cleaned up the examples. Any suggestions? 🙂

Comment (BalbinaK) [reply docs/outline-style.md:51]: Maybe it was supposed to be `Warp's focus style` then 😉

**Pull request merged**